solr/solrj/src/java/org/apache/solr/common/util/ModelConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.common.util;
+
+// TODO This probably shouldn't exist in a final product, but it's fine for a POC
```

Comment:
Out practice has been the magic string "nocommit" which we prevent merging but I see you use "TODO"... could go unnoticed as we don't have tooling to point these out.  I use "TODO" in a loose way but here you meant nocommit I think

Reply:
I am using it in a lot of places (but not always) as "NOCOMMIT".  It's a bad habit, I know 😬